cmd/ignite/run/rm.go

```diff
@@ -4,25 +4,55 @@ import (
 	"fmt"
 
 	api "github.com/weaveworks/ignite/pkg/apis/ignite"
+	"github.com/weaveworks/ignite/pkg/apis/ignite/scheme"
 	"github.com/weaveworks/ignite/pkg/operations"
 	"github.com/weaveworks/ignite/pkg/providers"
 )
 
+// RmFlags contains the flags supported by the remove command.
 type RmFlags struct {
-	Force bool
+	Force      bool
+	ConfigFile string
 }
 
 type rmOptions struct {
 	*RmFlags
 	vms []*api.VM
 }
 
-func (rf *RmFlags) NewRmOptions(vmMatches []string) (ro *rmOptions, err error) {
-	ro = &rmOptions{RmFlags: rf}
+// NewRmOptions creates and returns rmOptions with all the flags and VMs to be
+// removed.
+func (rf *RmFlags) NewRmOptions(vmMatches []string) (*rmOptions, error) {
+	ro := &rmOptions{RmFlags: rf}
+
+	// If config file is provided, use it to find the VM to be removed.
+	if len(rf.ConfigFile) != 0 {
```

Comment:
if it's provided, you mean `if len(rf.ConfigFile) > 0` ?

Reply:
That's right, I was checking if config file path isn't empty. In the lines of https://github.com/weaveworks/ignite/blob/v0.6.3/cmd/ignite/run/create.go#L73
Could have compared with `""` :slightly_smiling_face:  
Should I still change it?

Reply:
np.